rednotebook/util/markup.py

```diff
@@ -265,6 +269,17 @@ def _get_config(target, options):
         config['preproc'].append([r'\\\(\s*(.+?)\s*\\\)', r"BEGINMATH''\1''ENDMATH"])
         config['postproc'].append([r'BEGINMATH(.+)ENDMATH', r'$\1$'])
 
+        # Fix utf8 quotations - „, “ and ” cause problems compiling the latex document
+        config['postproc'].append([u'„', '"'])
+        config['postproc'].append([u'”', '"'])
+        config['postproc'].append([u'“', '"'])
```

Comment:
OK, let's leave this in then. Can you turn it into a single `config['postproc'].extend(...)` call, please?